urllib3/backends/sync_backend.py

```diff
@@ -2,11 +2,11 @@
 import select
 import socket
 import ssl
+from ..exceptions import _LoopAbort
```

Comment:
I think `urllib3.exceptions` is supposed to be a piece of public API... maybe it'd be better to move `LoopAbort` into `urllib3.backends._util`, which has random bits of code that's common across the backends? (And then we could import it in `urllib3/backends/__init__.py`. Maybe `_util` should be renamed `_common` or something.)